composer.json

```diff
@@ -21,14 +21,18 @@
     "yiisoft/yii2": "~2.0.0"
   },
   "require-dev": {
-    "phpunit/phpunit": "^6.0",
+    "phpunit/phpunit": "^6.0 || ^7.0 || ^8.0",
```

Comment:
In the `require-dev` section, I think we typically just specify a single version constraint (e.g. `^8.0`), and not use any `||`'s, since this is only used when developing/testing this repo locally. If it works with PHPUnit 8, then there's no reason to accept PHPUnit 6 or 7 any more.